### PR TITLE
Remove constraints

### PR DESCRIPTION
**Pull request by lmolkova**

### CODE REVIEW

**Comment by lmolkova** [comment]
We're moving over to [weaver](https://github.com/open-telemetry/weaver) to generate semantic conventions markdown and code and don't intend to release new version of build-tools.

So I'm going to close this PR for the time being.

File: semantic-conventions/src/tests/data/markdown/parameter_full/expected.md [hunk 1]
```diff
@@ -4,31 +4,7 @@
...
-| [`http.route`](http.md) | string | The matched route (path template). | `/users/:userID?` | `Recommended` | Experimental |
-| `http.scheme` | string | The URI scheme identifying the used protocol. | `http`; `https` | See below | Experimental |
-| `http.status_text` | string | [HTTP reason phrase](https://tools.ietf.org/html/rfc7230#section-3.1.2). | `OK` | `Recommended` | Experimental |
-| `http.target` | string | The full request target as passed in a HTTP request line or equivalent. | `/path/12314/?q=ddds#123` | See below | Experimental |
-| `http.url` | string | Full HTTP request URL in the form `scheme://host[:port]/path?query[#fragment]`. Usually the fragment is not transmitted over HTTP, but if it is known, it should be included nevertheless. | `https://www.foo.bar/search?q=OpenTelemetry#SemConv` | See below | Experimental |
-| `http.user_agent` | string | Value of the [HTTP User-Agent](https://tools.ietf.org/html/rfc7231#section-5.5.3) header sent by the client. | `CERN-LineMode/2.15 libwww/2.17b3` | `Recommended` | Experimental |
```
Comment (joaopgrassi) [inline semantic-conventions/src/tests/data/markdown/parameter_full/expected.md:18]: Hum.. why are these removed?

File: semantic-conventions/src/tests/semconv/model/test_correct_parse.py [hunk 2]
```diff
@@ -479,39 +443,6 @@ def test_extends(self):
...
-        self.assertEqual(len(semconv.models), 10)
-
-        faas_http = [s for s in semconv.models.values() if s.semconv_id == "faas.http"][
-            0
-        ]
-        expected = {
-            "id": "faas.http",
-            "prefix": "faas",
-            "extends": "faas",
-            "n_constraints": 2,
```
Comment (joaopgrassi) [inline semantic-conventions/src/tests/semconv/model/test_correct_parse.py:497]: shouldn't we remove just the constraint check? Or did I miss something?

**Review by jsuereth — APPROVED** [review]

File: semantic-conventions/src/tests/semconv/model/test_correct_parse.py (hunk 2 above)
Comment (lmolkova) [reply semantic-conventions/src/tests/semconv/model/test_correct_parse.py:497]: constraints have `include` mode which  adds all the stuff from included semconv into another one (similarly to `extends`).

since I'm removing them, included attributes dissapear from tests where constraints were used.

File: semantic-conventions/src/tests/data/markdown/parameter_full/expected.md (hunk 1 above)
Comment (lmolkova) [reply semantic-conventions/src/tests/data/markdown/parameter_full/expected.md:18]: see the other comment
